### DISCUSSION TITLE
[0.1300.0.0] Abandoned Outposts being destructible makes it easy to cheese

### DISCUSSION THREAD

**Discussion by Kaillera** [FakeFishGames/Barotrauma#5122] [Unstable feedback]
I think abandoned outpost doors and walls should be indestructible to prevent cheese with explosives or coilguns.

**Comment by ubertpendragon** [comment]
The fundamental problem here isn't that outposts are destructible, it's that you're encouraged to _mindlessly_ destroy them.
You basically have to kill everyone on the station, so you just point a railgun at the thing and poke a big hole into it. 

There needs to be an element that gets in the way of instantly destroying the outpost

So, instead of blowing up the reactor and killing everyone, let's say you need **to rescue a VIP**.
Put a handcuffed NPC in a cell that you need to bring back inside your sub:
- The random outpost generation will force you to first locate the NPC so you don't accidentally shoot it
- Breaching outpost walls will still be an option; avoid fighting by directly extracting the NPC from outside using a diving suit

This would feel better than straight up making walls magically indestructible

**Reply by mLuby** [reply]
Exactly. Players should be expected to complete any "destroy enemy station" missions by shelling it from the relative safety of their sub.

Rescue/kidnap VIP missions would encourage more judicious use of force. So would a "assassinate VIP" on an initially non-hostile station.

Another option would be to asses a "collateral damage" penalty for how much structure damage the station has at the end of the mission. It should be significant enough to make players consider whether to nuke the station for an easy, small payout or send a demolitions team for a riskier, higher reward.
